app/src/main/java/com/github/se/signify/ui/screens/home/ExerciseScreen.kt

```diff
@@ -91,6 +101,11 @@ fun ExerciseScreen(
           currentSentenceIndex = newSentenceIndex
         },
         onAllSentencesComplete = {
+          when (exerciseLevel.id) {
+            ExerciseLevelName.EASY -> statsViewModel.updateEasyExerciseStats()
+            ExerciseLevelName.MEDIUM -> statsViewModel.updateMediumExerciseStats()
+            ExerciseLevelName.HARD -> statsViewModel.updateHardExerciseStats()
+          }
```

Comment:
I have a small issue with this, currently the failure is not handled, the toast is shown and the state of the exercise is reset. I took a look at the StatsViewModel and I dont think there is a way for this screen to access the information on wether the call succeeded or not. I am not sure this should be the case and its out of scope for this PR, but maybe we should add a way for the ViewModel to expose the status of its calls to the repo.

Reply:
In fact, the only way to see if the update is done correctly is to go in the `MyStatsScreen` and see if the new value is displayed. For now the statsViewModel only has log.d and log.e for success and failure respectively, but it could be changed with other methods. Let me know how you think we should do this and I suggest to put it as a new task.

Reply:
Not sure yet but I think add a stateflow in the viewmodel with states like "idle","Loading","Success","Error" and the screen observes this to handle the outcome? I agree we can put this as a new task.